python/graphstorm/trainer/lp_trainer.py

```diff
@@ -210,11 +212,14 @@ def fit(self, train_loader, num_epochs,
                 break
 
         rt_profiler.save_profile()
-        print("Peak Mem alloc: {:.4f} MB".format(th.cuda.max_memory_allocated(device) / 1024 /1024))
+        print("Peak GPU Mem alloc: {:.4f} MB".format(th.cuda.max_memory_allocated(device) /
+                                                     1024 / 1024))
+        print("RAM memory used: {:.4f} MB".format(psutil.virtual_memory().used / 1024 / 1024))
         if self.rank == 0 and self.evaluator is not None:
             output = {'best_test_mrr': self.evaluator.best_test_score,
                        'best_val_mrr':self.evaluator.best_val_score,
-                       'peak_mem_alloc_MB': th.cuda.max_memory_allocated(device) / 1024 / 1024,
+                       'peak_GPU_mem_alloc_MB': th.cuda.max_memory_allocated(device) / 1024 / 1024,
+                       'RAM_mem_used': psutil.virtual_memory().used / 1024 / 1024,
```

Comment:
Is this message useful?
Can we get peak memory usage during training?

Reply:
I thought that printing used memory would be useful, but I could change it to eg. `resource.getrusage(resource.RUSAGE_SELF).ru_maxrss / 1024` 
from https://docs.python.org/3/library/resource.html#resource-usage which should return peak memory. Do you think this would fit better?